src/test/java/org/folio/rest/api/InstanceStorageTest.java

```diff
@@ -1522,14 +1332,6 @@ public void cannotFindIsbnWithInnerStringAndTruncation() {
     canSort("isbn = 552*");
   }
 
-  // Interesting Times has two ISBNs: 0552167541, 978-0-552-16754-3
```

Comment:
The test methods cannotFindIsbnWithTailString, cannotFindIsbnWithInnerStringAndTruncation, cannotFindIsbnInInvalidIsbn, cannotFindInvalidIsbnInIsbn no longer serve any purpose and should also been removed.